Header      HEADER
uint32      ID_SENDER
uint32      ID_RECEIVER
bool        FLAG

Task[]      MISSION
Task[]      ASSIGNED_MISSION

uint32[]    CURR_VERTEX
uint32[]    NEXT_VERTEX